frontend/src/components/navbar.css

```diff
@@ -24,23 +24,30 @@
   margin: 0rem;
 }
 
+.orange {
+  color: #f19b3b;
+}
+
+.pink {
+  block-size: inline;
+  margin: right;
+  color: #f05b75;
+}
+
 .nav a {
-  color: #ea953c;
+  color: #f19b3b;
   text-decoration: none;
   height: 100%;
   display: flex;
   align-items: center;
   padding: 0.25rem;
+  list-style: none;
 }
 
-.nav a:visited {
-  color: #ea953c;
-}
-
-.nav li.active {
+.active {
   background-color: #555;
```

Comment:
To fit the theme more,

I would think these would be more like

```css
.active {
    color: #eee;
    background-color: #f05b75;
}
```

So that it matches other things on the navbar. But this is up to you.